Pod/Classes/PubNub.swift

```diff
@@ -110,33 +110,28 @@ public extension PubNub {
                     return
                 }
 
-                results.insert(messages, atIndex: 0)
+                results.insert(messages, at: 0)
                 total += messages.count
 
-                if messages.count < PUBNUB_LIMIT || total >= limit || oldest.longLongValue <= newerThan?.longLongValue {
+                if messages.count < PUBNUB_LIMIT || total >= limit || oldest.int64Value <= (newerThan?.int64Value ?? 0) {
```

Comment:
Very not sure about this line @DavidSmackHigh

Reply:
This should be correct; seems like the longlongValue is a typeAlias for int64Value